repo/packages/P/portworx-data-science-engine/0/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.12",
+  "name": "portworx-data-science-engine",
+  "version": "1.3-1.0.0",
+  "scm": "https://github.com/mesosphere/jupyter-service",
+  "maintainer": "support@portworx.com",
+  "website": "https://docs.mesosphere.com/services/data-science-engine",
+  "description": "Portworx Data Science Engine provisions Jupyter Notebooks, an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text.\nUses include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.",
```

Comment:
The description needs to have some of the same language as the base package: https://github.com/portworx/jupyter-service/blob/master/universe/package.json#L9

Reply:
That's from master. The released tag had the above description. 
But looks like the actual package has description from master. I will update it.